src/taggle.js

```diff
@@ -268,17 +270,20 @@
                 return;
             }
 
+            if (!settings.onBeforeTagAdd(e, val)) {
```

Comment:
In my case i edit many «groups» of tags. Uniqueness of a tag must extend to all groups, so i can't use allowedTags. 
I have thousands of unique tags. Actualy, in my case they are not tags, but  "bag of words".